.github/workflows/generate-preview.yml

```
@@ -57,14 +81,27 @@ jobs:
               body: 'Please find a preview at: https://staging.nodejs.dev/${{ github.event.pull_request.number }}/'
             });
 
-      - name: Remove label
+      - name: Remove generating-preview label
         uses: actions/github-script@0.9.0
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             github.issues.removeLabels({
-              issue_number: context.pull_request.number,
+              issue_number: github.event.pull_request.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ['generating-preview']
+            })
+
+      - name: Add failure label if job broke
+        if: ${{ failure() }}
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.addLabels({
+              issue_number: github.event.pull_request.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              name: ['start-ci']
+              name: ['preview-failed']
```

I think the signature is a little different from the remove
```suggestion
              labels: ['preview-failed']
```